backend/inspirehep/mailing/config.py

```diff
@@ -19,8 +19,11 @@
 MAILTRAIN_URL = "https://lists.labs.inspirehep.net"
 MAILTRAIN_API_TOKEN = ""
 MAILTRAIN_JOBS_WEEKLY_LIST_ID = ""
-SENDGRID_API_KEY = 'CHANGE_ME'
+SENDGRID_API_KEY = "CHANGE_ME"
 """SendGrid api authentication key"""
 
 JOBS_DEADLINE_PASSED_SENDER_EMAIL = "jobs@inspirehep.net"
 """Email address for notifying expired jobs"""
+
+WEEKLY_JOBS_EMAIL_REDIS_KEY = "MAILTRAIN_KEY"
```

Comment:
I would pick something concrete here, like `jobs_weekly_email`.

Reply:
This was requested by Harun already to be "WEEKLY_JOBS_EMAIL_REDIS_KEY"

Reply:
I mean the value, so we don't have to override it later.

Reply:
:+1: